packages/ui-components/src/__tests__/TransactionManager.test.ts

```
@@ -138,41 +143,45 @@ describe('TransactionManager', () => {
+						fetchEntityFn: expect.any(Function),
 						isSuccess: expect.any(Function)
-					}
-				},
+					})
+				}),
 				expect.any(Function),
 				expect.any(Function)
 			);
 
 			const removeOrderCallArgs = vi.mocked(TransactionStore).mock.calls[0][0];
 			const removeOrderIsSuccessFn = removeOrderCallArgs.awaitSubgraphConfig!.isSuccess;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)